### PR TITLE
Feature/issues/253

### PR DESCRIPTION
**Pull request by Mizzick**

### CODE REVIEW

File: Extension/browser/chrome/lib/utils/file-storage.js [hunk 1]
```diff
@@ -23,162 +23,30 @@ var FS = exports.FS = {
 	LINE_BREAK: '\n',
 
 	readFromFile: function (path, callback) {
```
Comment (ameshkov) [inline Extension/browser/chrome/lib/utils/file-storage.js]: Working with local storage in "file-storage" class is simply misleading.

File: Extension/lib/utils/browser-utils.js [hunk 2]
```diff
@@ -100,6 +100,10 @@ var Utils = exports.Utils = {
         return Prefs.getBrowser() == "Chrome";
     },
 
+    isChromium: function () {
+        return Prefs.platform == 'chromium';
```
Comment (ameshkov) [inline Extension/lib/utils/browser-utils.js]: You'd better use this method everywhere instead of Prefs.platform == 'chromium'

Comment (Mizzick) [inline Extension/lib/utils/browser-utils.js]: But I can not use Utils everythere.

Comment (ameshkov) [inline Extension/lib/utils/browser-utils.js]: Ah, ok then.

**Pull request merged**